### DISCUSSION TITLE
iOS App won't display commits with no commit message

### DISCUSSION THREAD

**Discussion by cvengler** [community/community#7663] [Mobile]
Hello, I have a private repository that only consists of commits created with the following command:
```
git commit --allow-empty-message -m ''
```

When I try to view the list of commits within the mobile app I just get the following message: `There aren't any commits in this repository`.

I don't think that this behavior is desired especially when considering that the count of commits within the repository is displayed correctly and the web interface is able to show such commits too.

**Comment by brianlovin** [comment]
Thanks for the heads up on this @emilengler — I'm going to open an issue for our team to investigate this. We should probably fall back to the commit ID in the case of a missing commit message.

**Comment by brianlovin** [comment]
@emilengler are you using iOS or Android?

**Edit:** I was able to reproduce this on iOS. I've flagged it to our team to investigate 😊

**Reply by cvengler** [reply]
> @emilengler are you using iOS or Android?
> 
> 
> 
> **Edit:** I was able to reproduce this on iOS. I've flagged it to our team to investigate 😊 

I am on iOS. Sorry for not mentioning.
